### ISSUE TITLE
Localhost simulation build instructions do not work as documented

### ISSUE THREAD

**Issue by grempe** [dedis/cothority#261]
When following the instructions in the README things don't work as expected:

```
$ go get ./...
$ cd simul
$ go build
$ ./simul runfiles/sign_single.toml
```

It needs to be mentioned (for people who may not use Go everyday) that the repository needs to first be moved into the $GOPATH.

e.g. in my case I needed to clone the repo to:

```
~/go/src/github.com/cothority
```

This allowed the `go get ./...` and `go build` instructions to complete successfully.

However the final command references a file `runfiles/sign_single.toml` which no longer exists.  Picking other files from the `runtime` dir seems to work.  However it is not clear what one is accomplishing by running the simulations other than exercising tests on the code.  What is needed is higher level docs that describe the architecture, and how one would go about using any of this code in a real situation.

**Comment by Liamsi** [comment]
Thanks a lot for your feedback. Indeed, the file `sign_single.toml` doesn't exist anymore. I already changed this. Thanks for pointing out the other documentation shortcomings. Will be fixed (very) soon.

**Comment by ineiti** [comment]
Our goal is to move from simulation/research code to application-code during the next few weeks. As such I would very much appreciate if you could give us your use-case on how you would like to use collective signing:

- what application
- what language
- what you would like to have signed

**Comment by Liamsi** [comment]
> It needs to be mentioned (for people who may not use Go everyday) that the repository needs to first be moved into the $GOPATH.

I changed the instructions to be less ambiguous (`go get` clones the directory directly in the GOPATH). 
@ineiti @nikkolasg please review and merge: #265; best see how it looks here: https://github.com/dedis/cothority/tree/readme_started_rewrite) 

We really need to create a standalone application (similar to the one that existed before the SDA merge) as soon as possible and add a description on how-to run, deploy it etc. Similar to what @bford described recently.

@grempe If you have any further questions, feedback please let us know. Thanks again.

**Comment by ineiti** [comment]
Yes, we can discuss about how 'prototype' one once the checking is tested to run.

**Comment by ineiti** [comment]
Closed by #265
